src/Shared/Metrics/MetricsConstants.cs

```diff
@@ -10,4 +10,7 @@ internal static class MetricsConstants
 
     // Not based on a standard. Larger bucket sizes for longer lasting operations, e.g. HTTP connection duration. See https://github.com/open-telemetry/semantic-conventions/issues/336
     public static readonly IReadOnlyList<double> LongSecondsBucketBoundaries = [0.01, 0.02, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10, 30, 60, 120, 300];
+
+    // For Blazor/signalR sessions, which can last a long time.
+    public static readonly IReadOnlyList<double> VeryLongSecondsBucketBoundaries = [0.5, 1, 2, 5, 10, 30, 60, 120, 300, 600, 1500, 60*60, 2 * 60 * 60, 4 * 60 * 60];
```

Comment:
Is this used to register sessions in buckets within the histogram? 
Is there a limit on the number of values we can use here?
Could we use a more "human-friendly" set of values? Maybe something with more "resolution" in some buckets, for example, a few values for 0-1 minute (1,10,30,60), then 1-minute intervals up to 10, then 10 minutes interval up to the hour, then 2, 3, 6, 12, 24h?

Reply:
How we choose to bucketize things I imagine will impact the shape of the histogram/graph we use